aws-advanced-jdbc-wrapper-bundle/build.gradle.kts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("com.github.johnrengelman.shadow") version "6.0.0"
+}
+
+group = "software.amazon.jdbc"
+version = "2.3.2"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("software.amazon.awssdk:rds:2.22.13")
+    implementation("software.amazon.awssdk:sts:2.22.13")
+    implementation(project(":aws-advanced-jdbc-wrapper"))
+}
+
+tasks.shadowJar {
```

Comment:
NOTE: Was not able to find a way to shade/relocate all dependencies, and then add a exclusion. So i had to specify all dependencies (including transitive) and a couple more just in case.